webpack/assets/javascripts/react_app/actions/NotificationActions.js

```diff
@@ -1,18 +1,27 @@
 import API from '../API';
 import AppDispatcher from '../dispatcher';
-import {ACTIONS} from '../constants';
+import NotificationsStore from '../stores/NotificationsStore';
+import { ACTIONS, STATUS } from '../constants';
 
 const TIMER = 10000;
 
 export default {
   getNotifications(url) {
-    if (document.visibilityState === 'visible') {
+    if (document.visibilityState !== 'hidden' &&
```

Comment:
The Document.visibilityState read-only property returns the visibility of the document, that is in which context this element is now visible. It is useful to know if the document is in the background or an invisible tab, or only loaded for pre-rendering. Possible values are:

- 'visible' : the page content may be at least partially visible. In practice this means that the page is the foreground tab of a non-minimized window.
- 'hidden' : the page content is not visible to the user. In practice this means that the document is either a background tab or part of a minimized window, or the OS screen lock is active.
- 'prerender' : the page content is being prerendered and is not visible to the user (considered hidden for purposes of document.hidden). The document may start in this state, but will never transition to it from another value. Note: browser support is optional.
- 'unloaded' : the page is being unloaded from memory. Note: browser support is optional.

I made the change only because of the test since visibilityState is read-only and had a value of 'prerender' in my test. Alternatively, It is possible to use:

```js
(document.visibilityState === 'visible'  || document.visibilityState === 'prerender')
```